Source/DOH/fio.c

```diff
@@ -19,13 +19,13 @@
 static DOH *encodings = 0;	/* Encoding hash */
 
 /* -----------------------------------------------------------------------------
- * Writen()
+ * Written()
```

Comment:
This one is not a typo - it's "write n" (as is clear from the description).

Reply:
Reverted.